### PR TITLE
Update bogo_sort.c

### PR DESCRIPTION
**Pull request by riya-17**
#3181  Updated  bogo_sort.

I have resolved the issues created due to functions.

### CODE REVIEW

**Comment by riya-17** [comment]
Ok :)

**Comment by deepyadav442** [comment]
@riya-17  Can you directly merge your new code(as you are a member of Opengenus).I tried to fix sleep_sort.c file but again stuck with yours.
https://travis-ci.org/deepyadav442/cosmos/jobs/381977881  here are logs.
your last commit was unsuccessful as build failed.Do you know any other way of merging it directly.

**Comment by riya-17** [comment]
@deepyadav442  No, I don't know any other way of merging. If I come up with something I will surely do.

**Comment by ritwik12** [comment]
@deepyadav442 Correcting "sleep_sort.c" will not fix this as travis is failing for all three which are clang, gcc and g++. Check [here](https://travis-ci.org/OpenGenus/cosmos/builds/381748692?utm_source=github_status&utm_medium=notification). And sleep_sort.c is related with c;lang only.

**Comment by deepyadav442** [comment]
@ritwik12  I got it,but I was trying to debug files which were giving errors but as you said it will be better to change yml file because there are so many errors in different files.

File: code/sorting/src/bogo_sort/bogo_sort.c [hunk 1]
```diff
@@ -7,29 +7,17 @@
...
-    bogosort(num, 10);        
-    printf("The array after sorting is:");        
-    for (i = 0;i < 10;i++) {            
-        printf("%d\n", num[i]);        
-    } printf("\n");    
-}         
-int
-is_sorted(int *a, int n)    
+        
+int is_sorted(int *a, int n)
```
Comment (christianbender) [inline code/sorting/src/bogo_sort/bogo_sort.c]: The type goes on is own line. See https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c

File: code/sorting/src/bogo_sort/bogo_sort.c [hunk 2]
```diff
@@ -38,10 +26,21 @@ shuffle(int *a, int n)
...
+
+void bogosort(int *a, int n)    
 {        
     while ( !is_sorted(a, n) )             
     shuffle(a, n);    
 }
+
+int main()    
+{        
+    int num[10]={1, 4, 7, 5, 9, 2, 6, 3, 8, 0};
```
Comment (christianbender) [inline code/sorting/src/bogo_sort/bogo_sort.c]: You can write ```num[]``` instead of ```num[10]```.

File: code/sorting/src/bogo_sort/bogo_sort.c [hunk 3]
```diff
@@ -38,10 +26,21 @@ shuffle(int *a, int n)
...
     }    
-}    
-void
-bogosort(int *a, int n)    
+}   
+
+void bogosort(int *a, int n)    
 {        
     while ( !is_sorted(a, n) )             
     shuffle(a, n);
```
Comment (christianbender) [inline code/sorting/src/bogo_sort/bogo_sort.c]: Indent this line.

**Comment by riya-17** [comment]
@christianbender thankyou Sir for your suggestion and time

**Pull request closed without merging**